backend/functions/lib/cosmos-humble-objects/fake.cases.cosmos-client-humble.ts

```diff
@@ -1,4 +1,7 @@
+import { CaseAssignmentHistory } from '../adapters/types/case.history';
 import { HumbleClient } from '../testing/mock.cosmos-client-humble';
 import { TransferIn, TransferOut } from '../use-cases/orders/orders.model';
 
-export default class FakeCasesCosmosClientHumble extends HumbleClient<TransferIn | TransferOut> {}
+export default class FakeCasesCosmosClientHumble extends HumbleClient<
+  TransferIn | TransferOut | CaseAssignmentHistory
```

Comment:
This maybe should just be CaseHistory now. We shouldn't be exporting CaseAssignmentHistory anymore?

Reply:
Yeah.  Agreed.